### ISSUE TITLE
Doesn't report Read / Write access for current word under the cursor

### ISSUE THREAD

**Issue by jonathanding** [microsoft/pylance-release#42]
## Environment data

-   Language Server version: 2020.6.1
-   OS and version: Windows 10 Enterprise, Build 18363
-   Python version (& distribution if applicable, e.g. Anaconda): Python 3.8.3 64bit

## Expected behaviour

According to https://code.visualstudio.com/api/references/theme-color#editor-colors, the language server should report read/write access for current variable (the word under the cursor), so that theme could assign them with different colors. 

This is captured from the official documentation of VSCode.

<img src="https://user-images.githubusercontent.com/1069667/86452338-bc271c80-bd4e-11ea-9965-1f5eee28370b.jpg" width="600">

This does work as expected if using old "Microsoft" language server.  

As shown below, the variable "test" is under cursor, and it's write access so rendered as magenta, while the other occurrence of "test" is read access and rendered as blue.

<img src="https://user-images.githubusercontent.com/1069667/86452102-6ce0ec00-bd4e-11ea-8321-fbabdc3464cf.jpg" width="400">

It's magenta and blue because I configured this in my settings as below.

<img src="https://user-images.githubusercontent.com/1069667/86452368-c77a4800-bd4e-11ea-868e-fe40f268d665.jpg" width="400">

## Actual behaviour
However, after switch to Pylance, it doesn't work any more. You would see both occurrences are rendered as green -- this is because language sever doesn't report the access type and it falls back to use the color defined for "selectionHighlightBackground" in my settings above.

<img src="https://user-images.githubusercontent.com/1069667/86452410-d82abe00-bd4e-11ea-8240-2ef1ca77f21b.jpg" width="400">

## Logs

```
[Info  - 4:49:07 PM] Pylance language server 2020.6.1 starting
[Info  - 4:49:07 PM] Server root directory: c:\Users\jding4\.vscode\extensions\ms-python.vscode-pylance-2020.6.1\server
[Info  - 4:52:00 PM] No configuration file found.
[Info  - 4:52:00 PM] Setting pythonPath for service "fda": "C:\Python\python.exe"
[Info  - 4:52:00 PM] stubPath c:\workspace\fda\typings is not a valid directory.
[Info  - 4:52:00 PM] Assuming Python version 3.8
[Info  - 4:52:00 PM] Assuming Python platform Windows
[Info  - 4:52:00 PM] Searching for source files
[Info  - 4:52:00 PM] Found 1 source file
[BG] analyzing: c:\workspace\fda\test.py ...
[BG]   parsing: c:\workspace\fda\test.py (18ms)
[BG]   parsing: c:\Users\jding4\.vscode\extensions\ms-python.vscode-pylance-2020.6.1\server\typeshed-fallback\stdlib\2and3\builtins.pyi (470ms)
[BG]   binding: c:\Users\jding4\.vscode\extensions\ms-python.vscode-pylance-2020.6.1\server\typeshed-fallback\stdlib\2and3\builtins.pyi (122ms)
[BG]   binding: c:\workspace\fda\test.py (1ms)
[BG]   checking: c:\workspace\fda\test.py ...
[BG]     parsing: c:\Users\jding4\.vscode\extensions\ms-python.vscode-pylance-2020.6.1\server\typeshed-fallback\stdlib\3\typing.pyi (88ms)
[BG]     binding: c:\Users\jding4\.vscode\extensions\ms-python.vscode-pylance-2020.6.1\server\typeshed-fallback\stdlib\3\typing.pyi (33ms)
[BG]     parsing: c:\Users\jding4\.vscode\extensions\ms-python.vscode-pylance-2020.6.1\server\typeshed-fallback\stdlib\2and3\_typeshed\__init__.pyi (17ms)
[BG]     binding: c:\Users\jding4\.vscode\extensions\ms-python.vscode-pylance-2020.6.1\server\typeshed-fallback\stdlib\2and3\_typeshed\__init__.pyi (2ms)
[BG]   checking: c:\workspace\fda\test.py (189ms)
[BG] analyzing: c:\workspace\fda\test.py (802ms)
[FG] parsing: c:\workspace\fda\test.py (47ms)
[FG] parsing: c:\Users\jding4\.vscode\extensions\ms-python.vscode-pylance-2020.6.1\server\typeshed-fallback\stdlib\2and3\builtins.pyi (1031ms)
[FG] binding: c:\Users\jding4\.vscode\extensions\ms-python.vscode-pylance-2020.6.1\server\typeshed-fallback\stdlib\2and3\builtins.pyi (293ms)
[FG] binding: c:\workspace\fda\test.py (5ms)
[FG] parsing: c:\Users\jding4\.vscode\extensions\ms-python.vscode-pylance-2020.6.1\server\typeshed-fallback\stdlib\3\typing.pyi (165ms)
[FG] binding: c:\Users\jding4\.vscode\extensions\ms-python.vscode-pylance-2020.6.1\server\typeshed-fallback\stdlib\3\typing.pyi (90ms)
[FG] parsing: c:\Users\jding4\.vscode\extensions\ms-python.vscode-pylance-2020.6.1\server\typeshed-fallback\stdlib\2and3\_typeshed\__init__.pyi (29ms)
[FG] binding: c:\Users\jding4\.vscode\extensions\ms-python.vscode-pylance-2020.6.1\server\typeshed-fallback\stdlib\2and3\_typeshed\__init__.pyi (3ms
```

## Code Snippet / Additional information

```python
def f():
    test = "Write access for the variable"
    print(test, "Read access for the variable"
```

**Comment by huguesv** [comment]
Looks like it is this LSP request which provides this information in MPLS, but is not supported yet in Pylance: https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_documentHighlight

**Comment by erictraut** [comment]
I implemented this support, and it will be included in an upcoming version of Pylance.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.7.0, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#202070-9-july-2020
